composer.json

```diff
@@ -14,18 +14,28 @@
 			"role": "Developer"
 		}
 	],
+	"repositories": [
```

Comment:
I will create a follow-up issue for this when this PR is merged. We'll need to switch back to the main WP Acceptance repo after Felipe's PR is merged.

Reply:
@johnwatkins0 FYI that WPA has been updated from Felipe's PR.